### ISSUE TITLE
Color Contrast in cookie settings box is not sufficient

### ISSUE THREAD

**Issue by reetagithub** [DSpace/dspace-angular#2668]
**Describe the bug**
In the "We collect and process your personal information..." box, there is green text on the black background. The contrast between these two is not sufficient: accessibility requirements say it should be 4.5:1, but it is only 3.27:1.

**To Reproduce**
Use WAVE on the page, or fill https://webaim.org/resources/contrastchecker/

**Comment by tdonohue** [comment]
**PR #3039 has partially fixed this issue (for 7.6.2 and 8.0).**  That PR updates the styles of the initial Klaro popup window to have proper color contrast.  Here's the new look:
![initialpopup](https://github.com/DSpace/dspace-angular/assets/483997/0432d8c4-b3ce-4d0d-91e3-13f82b0d7465)

However, if you click on "Customize" in that popup, you'll see a larger list of all possible settings.  This larger popup **still has color contrast issues** (a lot of dark gray text on a black background) as shown in this screenshot:
![FullSettings](https://github.com/DSpace/dspace-angular/assets/483997/a59a2e57-3d0a-4565-9c6d-aceaf24ce2b1)

**Comment by Andrea-Guevara** [comment]
@tdonohue Hi, I'd like to take this activity. I understood that the problem was only the gray color of some texts. Is that right?

**Comment by tdonohue** [comment]
@Andrea-Guevara : I'm not exactly sure if the gray color is the **only issue**.  I was mentioning it as an example.  The primarily problem is that this page in the popup **fails** when analyzed by an accessibility scanner (e.g. WAVE or the "axe DevTools" plugin in Chrome).

https://private-user-images.githubusercontent.com/483997/339895603-a59a2e57-3d0a-4565-9c6d-aceaf24ce2b1.png

So, to fix this ticket would require that all pages within the Klaro popup to succeed accessibility scanning. 

To reproduce:
* Visit https://sandbox.dspace.org in an Incognito browser
* Klaro cookie popup should appear.  Click on "Customize" to bring up the screen above
* Expand all sections on that screen.  
* Run an accessibility scanner (either WAVE or "axe DevTools).  Both will report 5-10 color contrast issues.  Those are what need to be fixed.

I'll assign this to you @Andrea-Guevara .  But, if you change your mind, let me know and we can find a different volunteer.

**Comment by Andrea-Guevara** [comment]
Hello, good afternoon.

I did some tests and managed to reduce the accessibility problems of the cookis configuration box.

However, I had to use "!important" in some parts of the code for the css changes to take effect. Is this a problem?

I look forward to your feedback.
![issue-2668](https://github.com/DSpace/dspace-angular/assets/101608067/d6610b47-7542-48ab-b581-0e3ad26c2ed4)

**Comment by Andrea-Guevara** [comment]
Good afternoon @tdonohue! I've made a PR so you can see the code.
https://github.com/DSpace/dspace-angular/pull/3140

I look forward to your feedback

**Comment by Andrea-Guevara** [comment]
Good afternoon @tdonohue! I made a new PR updated to DSpace version 8 and the pull request has now gone to the dspace-8_x branch.

Is that right? 
If not, let me know! I'll be waiting.

PR updated: https://github.com/DSpace/dspace-angular/pull/3328

**Comment by tdonohue** [comment]
Closing, fixed by #3328
